service/src/main/java/aero/minova/cas/ControllerExceptionHandler.java

```diff
@@ -190,23 +195,37 @@ private Table handleGenericErrorMessage(Table outputTable, String errorMessage)
 
 	/**
```

Comment:
Bitte dies in [aero.minova.cas](https://github.com/minova-afis/aero.minova.cas)/[api](https://github.com/minova-afis/aero.minova.cas/tree/master/api)/[doc](https://github.com/minova-afis/aero.minova.cas/tree/master/api/doc)/[adoc](https://github.com/minova-afis/aero.minova.cas/tree/master/api/doc/adoc)/api-procedure.adoc dokumentieren.